definitions/ext-ups/kentik-ups-dashboard.json

```diff
@@ -2,195 +2,224 @@
-					}
-				},
+				  "accountId": 0,
+				  "query": "FROM Metric SELECT if(latest(upsBatteryStatus) = 'batteryNormal', 0, 1) FACET upsBatteryStatus WHERE provider = 'kentik-ups'"
+				}
+			  ],
+			  "platformOptions": {
+				"ignoreTimeRange": false
+			  },
+			  "thresholds": [
 				{
-					"visualization": {
-						"id": "viz.billboard"
-					},
-					"layout": {
-						"column": 5,
-						"row": 4,
-						"height": 3,
-						"width": 2
-					},
-					"title": "Battery Capacity Status (1 == Unknown, 2 == Normal, 3 == Low, 4 == Depleted)",
-					"rawConfiguration": {
-						"dataFormatters": [],
-						"nrqlQueries": [
-							{
-								"accountId": 0,
-								"query": "FROM Metric SELECT latest(kentik.snmp.upsBatteryStatus) AS 'Battery Status' WHERE provider = 'kentik-ups'"
-							}
-						],
-						"thresholds": [
-							{
-								"alertSeverity": "WARNING",
-								"value": 3
-							},
-							{
-								"alertSeverity": "CRITICAL",
-								"value": 4
-							}
-						]
-					}
+				  "alertSeverity": "WARNING",
+				  "value": 0.1
 				},
 				{
-					"visualization": {
-						"id": "viz.billboard"
-					},
-					"layout": {
-						"column": 7,
-						"row": 4,
-						"height": 3,
-						"width": 2
-					},
-					"title": "Active Alarm Conditions (Count)",
-					"rawConfiguration": {
-						"dataFormatters": [],
-						"nrqlQueries": [
-							{
-								"accountId": 0,
-								"query": "FROM Metric SELECT latest(kentik.snmp.upsAlarmsPresent) AS 'Active Alarm Conditions' WHERE provider = 'kentik-ups'"
-							}
-						],
-						"thresholds": [
-							{
-								"alertSeverity": "WARNING",
-								"value": 0.5
-							},
-							{
-								"alertSeverity": "CRITICAL",
-								"value": 0.9
-							}
-						]
-					}
-				},
+				  "alertSeverity": "CRITICAL",
+				  "value": 0.2
+				}
+			  ]
+			}
+		  },
+		  {
+			"title": "Battery Summary",
+			"layout": {
+			  "column": 5,
+			  "row": 1,
+			  "width": 3,
+			  "height": 4
+			},
+			"visualization": {
+			  "id": "viz.billboard"
+			},
+			"rawConfiguration": {
+			  "facet": {
+				"showOtherSeries": false
+			  },
+			  "nrqlQueries": [
+				{
+				  "accountId": 0,
+				  "query": "FROM Metric SELECT latest(kentik.snmp.upsEstimatedChargeRemaining) AS 'Battery Charge (%)', latest(upsTestResultsDetail) AS 'Latest Test Results' WHERE provider = 'kentik-ups'"
+				}
+			  ],
+			  "platformOptions": {
+				"ignoreTimeRange": false
+			  }
+			}
+		  },
+		  {
+			"title": "Output Load Capacity %",
+			"layout": {
+			  "column": 8,
+			  "row": 1,
+			  "width": 5,
+			  "height": 4
+			},
+			"visualization": {
+			  "id": "viz.line"
+			},
+			"rawConfiguration": {
+			  "facet": {
+				"showOtherSeries": false
+			  },
+			  "legend": {
+				"enabled": true
+			  },
+			  "nrqlQueries": [
+				{
+				  "accountId": 0,
+				  "query": "FROM Metric SELECT latest(kentik.snmp.upsOutputPercentLoad) AS 'Output Load %' WHERE provider = 'kentik-ups' TIMESERIES 5 MINUTES"
```

Comment:
I would remove the 5 MINUTES from timeseries or set it to MAX as timerange can be changed by user

Reply:
same comment from the APC dashboard